File: dynamic-datasource-builder/src/main/java/com/bip/model/ConnectionModel.java
package com.bip.model;

import com.bip.entity.Connection;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class ConnectionModel {
	private String dataSourceName;
	private String url;
	private String username;
	private String password;
	private String type;

	public Connection _toConvertConnectionEntity(){
		Connection entity = new Connection();
		entity.setDataSourceName(this.dataSourceName);
		entity.setUrl(this.url);
		entity.setUsername(this.username);
		entity.setPassword(this.password);
		entity.setType((this.type));
		return entity;
	}
}

File: data-source-builder-client/src/app/model/connection.model.ts
export class ConnectionModel{
    dataSourceName: string
    url: string;
    username: string;
    password: string;
    type: string;
    constructor(dataSourceName: string, url : string, username: string, password: string, type:string){
        this.dataSourceName = dataSourceName;
        this.url = url;
        this.password = password;
        this.username = username;
        this.type = type;
    }
}